### ISSUE TITLE
Is the wiki outdated?

### ISSUE THREAD

**Issue by GhbSmwc** [greasemonkey/greasemonkey#3147]
This: https://wiki.greasespot.net/GM.registerMenuCommand
This is what I literally have in my script to see if it does something in the GM menu:
```
// ==UserScript==
// @name     Menu Test
// @version  1
// @grant    none
// ==/UserScript==
function hello() {console.log(console.log("Hello world!"))}
GM.registerMenuCommand("Hello, world!", hello, "h");
```
And it doesn't shows up on the menu.

Also, the syntax shows a dot: ``GM.registerMenuCommand( caption, commandFunc, accessKey )`` but the examples shows them as underscores: ``GM_registerMenuCommand("Hello, world!", hello, "h");``

**Comment by Cerberus-tm** [comment]
The underscore is probably outdated.

Have you tried it while granting the GM.registerMenuCommand permission? I think you need to @grant most GM. things in the introductory block.

**Comment by GhbSmwc** [comment]
No, how do I do that? What should be after the @grant?

**Comment by Cerberus-tm** [comment]
Yes, exactly like that! It is needed when you use special things that normal scripts on a page cannot do: things specifically empowered by the Greasemonkey extension, as opposed to normal Javascript. It is meant for security, so that you can always see at the top of a script when it has permission to do stuff that might penetrate normal privacy/security barriers.

**Comment by GhbSmwc** [comment]
Yay, it is working!

**Comment by Cerberus-tm** [comment]
Excellent!

**Comment by arantius** [comment]
Thanks for pointing this out, the wiki is fixed.  (As Cerberus points out, `GM_` style was for Greasemonkey 3 and earlier, while version 4 switched to the `GM.` style.)
